rest/data/interface.go

```diff
@@ -43,7 +43,7 @@ type Connector interface {
 
 	// FindTaskById is a method to find a specific task given its ID.
 	FindTaskById(string) (*task.Task, error)
-	FindOldTasksByID(string) ([]task.Task, error)
+	FindOldTasksByIDWithDisplayTasks(string) ([]task.Task, error)
```

Comment:
I don't think this needed to be renamed. It's still accurate and the new name is a mouthful

Reply:
Disagree, because there is also a method in model/tasks called FindOld that fetches old tasks without display tasks. This confusion is what led to me making this mistake in the first place.